ios/FluentUI/Core/FluentUIFramework.swift

```diff
@@ -88,8 +70,15 @@ public class FluentUIFramework: NSObject {
         // UIToolbar
         let toolbar = UIToolbar.appearance()
         toolbar.isTranslucent = false
-        toolbar.barTintColor = Colors.Toolbar.background
-        toolbar.tintColor = Colors.Toolbar.tint
+
+        let aliasTokens = AliasTokens()
+        if let fluentTheme = fluentTheme {
+            toolbar.barTintColor = UIColor(dynamicColor: fluentTheme.aliasTokens.colors[.background3])
+            toolbar.tintColor = UIColor(dynamicColor: fluentTheme.aliasTokens.colors[.foreground3])
+        } else {
+            toolbar.barTintColor = UIColor(dynamicColor: aliasTokens.colors[.background3])
+            toolbar.tintColor = UIColor(dynamicColor: aliasTokens.colors[.foreground3])
+        }
```

Comment:
I think this could probably be:
```
let aliasTokens: AliasTokens
if let fluentTheme = fluentTheme {
  aliasTokens = fluentTheme.aliasTokens
} else {
  aliasTokens = AliasTokens()
}
```
And then you could move setting the colors to outside the `if let`, and also get `progressViewAppearance.trackTintColor` from the theme.

Reply:
nit: Is this also equivalent to `let aliasTokens = fluentTheme.aliasTokens ?? AliasTokens()`?

Reply:
Ah, yeah, that's probably actually better.

Reply:
Done! Good teamwork :D